#! /bin/sh /usr/share/dpatch/dpatch-run
## 85_URI_fixup.dpatch by  <glandium@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Enable keyword lookup by default in URI fixup. Temporary until Galeon 
## DP: and Kazehakase are fixed

@DPATCH@

--- xulrunner/docshell/base/nsDocShell.cpp
+++ xulrunner/docshell/base/nsDocShell.cpp
@@ -2864,6 +2864,8 @@
     rv = NS_NewURI(getter_AddRefs(uri), uriString);
     if (uri) {
         aLoadFlags = aLoadFlags & ~LOAD_FLAGS_ALLOW_THIRD_PARTY_FIXUP;
+    } else {
+        aLoadFlags |= LOAD_FLAGS_ALLOW_THIRD_PARTY_FIXUP;
     }
     
     if (sURIFixup) {
